test/extended/util/jenkins/ref.go

```diff
@@ -413,12 +405,8 @@ func SetupSnapshotImage(envVarName, localImageName, snapshotImageStream string,
 
 	} else {
 		if snapshotImagePresent {
-			ginkgolog("")
-			ginkgolog("")
-			ginkgolog("IMPORTANT: You have a local OpenShift jenkins snapshot image, but it is not being used for testing.")
-			ginkgolog("In order to target your local image, you must set %s to some value before running extended tests.", envVarName)
-			ginkgolog("")
-			ginkgolog("")
+			fmt.Fprintf(g.GinkgoWriter, "\n\nIMPORTANT: You have a local OpenShift jenkins snapshot image, but it is not being used for testing.")
+			fmt.Fprintf(g.GinkgoWriter, "In order to target your local image, you must set %s to some value before running extended tests.\n\n", envVarName)
```

Comment:
why not e2e.logf for these?

Reply:
e2e.Logf prints an "INFO" at the beginning of the line, and i thought it looked weird in front of these couple of messages, but I can add it if you want.

Reply:
i'll leave it up to @gabemontero since it's primarily his test, but i'd sort of be inclined for consistency over getting fancy w/ the formatting of different output in this.

Reply:
updated for consistency